### DISCUSSION TITLE
Does this code need reviewing?

### DISCUSSION THREAD

**Discussion by smithse** [dl9rdz/rdz_ttgo_sonde#496] [Developers corner]
In reference to a previous issue [https://github.com/dl9rdz/rdz_ttgo_sonde/issues/483](url), does this code need to be reviewed?

This entry in the config.txt file: [https://github.com/dl9rdz/rdz_ttgo_sonde/blob/963f44df2e85c8a6a52aef82ffcf807bbe61bec2/RX_FSK/data/config.txt#L111](url) seems to be reversed here: [https://github.com/dl9rdz/rdz_ttgo_sonde/blob/963f44df2e85c8a6a52aef82ffcf807bbe61bec2/RX_FSK/src/Sonde.cpp#L331](url)

I am trying to understand the code, and would I be correct in saying that the default settings are initially written from Sonde.cpp into the config.txt file if it's missing??

**Comment by dl9rdz** [comment]
First the default values are set, and then replaced with values from the config file (if it exists / contains those values).
Yes this needs to be reviewed. Will do so later

**Reply by smithse** [reply]
Thanks for the prompt response. I will do some testing locally. Happy New Year

**Comment by smithse** [comment]
Looks like you beat me to it! 🥇
